model_compression_toolkit/core/common/mixed_precision/resource_utilization_tools/ru_methods.py

```diff
@@ -87,6 +90,88 @@ def weights_size_utilization(mp_cfg: List[int],
     return np.array(weights_memory)
 
 
+def calc_graph_cuts(graph: Graph) -> List[Cut]:
+    """
+    Calculate graph activation cuts.
+    Args:
+        graph: A graph object to calculate activation cuts on.
+
+    Returns:
+        A list of activation cuts.
+
+    """
+    memory_graph = MemoryGraph(deepcopy(graph))  # TODO maxcut: do we need the deepcopy?
+    _, _, cuts = compute_graph_max_cut(memory_graph)
+
+    if cuts is None:
+        return None
+    else:
+        # filter empty cuts and cuta that contain nodes with activation quantization disabled.
+        filtered_cuts = []
+        for cut in cuts:
+            if len(cut.mem_elements.elements) > 0 and any(
+                    [graph.find_node_by_name(e.node_name)[0].has_activation_quantization_enabled_candidate()
+                     for e in cut.mem_elements.elements]):
+                filtered_cuts.append(cut)
+        return filtered_cuts
+
+
+def activation_maxcut_size_utilization(mp_cfg: List[int],
+                                       graph: Graph,
+                                       fw_info: FrameworkInfo,
+                                       fw_impl: FrameworkImplementation,
+                                       cuts: List[Cut] = None) -> np.ndarray:
+    """
+    Computes a resource utilization vector with the respective output memory max-cut size for activation
+    nodes, according to the given mixed-precision configuration.
+
+    Args:
+        mp_cfg: A mixed-precision configuration (list of candidates index for each configurable node)
+        graph: Graph object.
+        fw_info: FrameworkInfo object about the specific framework (e.g., attributes of different layers' weights to quantize)
+            (not used in this method).
+        fw_impl: FrameworkImplementation object with specific framework methods implementation(not used in this method).
+        cuts: a list of graph cuts (optional. if not provided calculated locally).
+
+    Returns: A vector of node's cut memory sizes.
+    Note that the vector is not necessarily of the same length as the given config.
+
+    """
+    activation_cut_memory = []
+    if len(mp_cfg) == 0:
```

Comment:
suggestion for minor change:
- Change the comment for an informative one that states that non-configurable nodes are not included in the cuts memory.
- init activation_cut_memory  with a cut of size 0
- remove 'if'
- add an assertion that node_idx < len(mp_cfg)